docs/testing/unit-testing.md

```diff
@@ -1071,6 +1071,10 @@ Jest incorporates most of the features of Sinon with a slimmer API:
 - `jest.spyOn(object, method)` can be used in place of `sinon.spy(object, method)` or `sinon.stub(object, method)` (with the caveat that the method being spied upon will still be called by default).
 - `jest.useFakeTimers()` can be used in place of `sinon.useFakeTimers()` (though note that Jest's "clock" object had fewer features than Sinon's prior to Jest v29.5).
 
+## Avoid general manual mocks:
+
+According to Jest's documentation `Manual mocks are defined by writing a module in a __mocks__/ subdirectory immediately`. These types of mocks are automatically picked up by Jest for all tests. We should be very careful when writing this types of mocks as they will be shared across all tests (including UI integration tests).
```

Comment:
I think it could be helpful to extend this suggestion to the `ui-integration-testing.md`. possibly under "## Don't Mock UI Components". We could also append "(including unit tests)." to the end